Library/Homebrew/extend/os/linux/extend/pathname.rb

```diff
@@ -0,0 +1,13 @@
+class Pathname
+  # @private
+  def elf?
+    # List of file signatures: https://en.wikipedia.org/wiki/List_of_file_signatures
```

Comment:
should I add a pointer to ` /usr/share/file/magic` directory and/or `man magic`?

Reply:
One reference is good enough. The ELF article on Wikipedia is a better reference though:
https://en.wikipedia.org/wiki/Executable_and_Linkable_Format#File_header